libsplinter/src/biome/refresh_tokens/store/diesel/models.rs

```diff
@@ -0,0 +1,33 @@
+// Copyright 2018-2020 Cargill Incorporated
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use super::schema::refresh_tokens;
+
+#[cfg(feature = "biome-refresh-tokens")]
```

Comment:
This and the one below are unnecessary as it is already checked withing the biome:: module.

Reply:
Updated